src/stores/events/EventsDataStore.ts

```diff
@@ -284,14 +292,34 @@ export default class EventsDataStore {
 					if (cachedRootNode) {
 						rootNode = cachedRootNode;
 					} else {
-						rootNode = getErrorEventTreeNode(rootNode?.parentId || parentId);
+						rootNode = unknownRoot;
+						if (currentParentId) {
+							this.parentChildrensMap.set(unknownRoot.eventId, [
+								...(this.parentChildrensMap.get(unknownRoot.eventId) || []),
+								...(this.parentChildrensMap.get(currentParentId) || []),
+							]);
+							this.parentChildrensMap.delete(currentParentId);
+							this.rootEventIds = [...this.rootEventIds, currentParentId];
```

Comment:
why currentParentId should be in `rootEventIds`? isn't that an uknown event that supposed to be in `unknownRoot`

Reply:
We contains only real ids eventDataStore.rootEventIds for correct target node resolving

Reply:
I think that's confusing, i think there should be only real root event id's. There might be some diffucilties with resolving and expanding target node if it's inside the unknown event, but i don't think that's a good solution to it